libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -1979,19 +1971,32 @@ CK_RV prvCreatePrivateKey( CK_ATTRIBUTE_PTR pxTemplate,
                     xMbedContext.pk_ctx = pxKeyPair;
                     mbedtls_ecp_keypair_init( pxKeyPair );
                     mbedtls_ecp_group_init( &pxKeyPair->grp );
+
                     /*/ * At this time, only P-256 curves are supported. * / */
-                    mbedtls_ecp_group_load( &pxKeyPair->grp, MBEDTLS_ECP_DP_SECP256R1 );
+                    lMbedTLSReturn = mbedtls_ecp_group_load( &pxKeyPair->grp, MBEDTLS_ECP_DP_SECP256R1 );
+
+                    if( lMbedTLSReturn != 0 )
+                    {
+                        PKCS11_PRINT( ( "mbedTLS ECP curve load failed with error %s : %s \r\n",
+                                        mbedtlsHighLevelCodeOrDefault( lMbedTLSReturn ),
+                                        mbedtlsLowLevelCodeOrDefault( lMbedTLSReturn ) ) );
+                        xResult = CKR_FUNCTION_FAILED;
+                    }
                 }
                 else
                 {
                     xResult = CKR_HOST_MEMORY;
                 }
             }
 
-            xResult = prvCreateEcPrivateKey( &xMbedContext,
-                                             &pxLabel,
-                                             pxTemplate,
-                                             ulCount );
+            /* Stopped here */
```

Comment:
What does this comment mean?

Reply:
Need to remove, this was for my internal work

Reply:
Removed